src/pages/organisationUnits/list/useOrganisationUnits.tsx

```diff
@@ -60,7 +60,7 @@ export const useFilteredOrgUnits = ({
 export type ParentIdToPages = Record<string, number[]>
 
 type UsePaginatedChildrenOrgUnitsOptions = {
-    parentIds: string[]
+    parentIds: Record<string, boolean>
```

Comment:
Decided to change the API here, because then we can reuse the object as is instead of transforming and having to memo the array